src/actions/actions.ts

```diff
@@ -1960,6 +1960,11 @@ class ActionDeleteChar extends BaseCommand {
 }
 
 @RegisterAction
+class ActionDeleteCharWithDeleteKey extends ActionDeleteChar {
+  keys = ["<delete>"];
+}
```

Comment:
I thought `N<Del>` to delete the next 5 chars should be the expected behavior.

So, in Normal Mode, is this

`<Del>` -> removes next char
`N<Del>` -> do nothing

what I should do instead?